scripts/tekton/collections-build-task.yaml

```diff
@@ -0,0 +1,202 @@
+apiVersion: tekton.dev/v1alpha1
```

Comment:
Following the common theme can we remove collections and kabanero references.

Reply:
Done